backend/src/middleware/permissionsMiddleware.js

```diff
@@ -157,6 +157,8 @@ const permissions = shield(
       User: or(noEmailFilter, isAdmin),
       isLoggedIn: allow,
       Badge: allow,
+      PostsEmotionsCountByEmotion: allow,
+      PostsEmotionsByCurrentUser: allow,
     },
     Mutation: {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-06T20:23:00Z" title="Tuesday, August 6th 2019, 10:23:00 pm +02:00">Aug 6, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -27,6 +27,12 @@ const onlyYourself = rule({
   return context.user.id === args.id
 })
 
+const isMyEmotion = rule({
+  cache: 'no_cache',
+})(async (parent, args, context, info) => {
+  return context.user.id === args.from.id
```


Eww - implement your own custom resolvers and you don't have to check that anymore.

![eww](https://media.giphy.com/media/122pEb8wGOZviM/giphy.gif)

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-07T05:01:20Z" title="Wednesday, August 7th 2019, 7:01:20 am +02:00">Aug 7, 2019</time>_

---


good catch